geode-core/build.gradle

```diff
@@ -375,6 +375,7 @@ dependencies {
 
 
   distributedTestImplementation(project(':geode-gfsh'))
+  distributedTestImplementation(project(':geode-wan'))
```

Comment:
Since `geode-wan` was needed for these new distributed tests perhaps that is a sign the tests belong in `geode-wan`?

Reply:
You are right! I will move this test to geode-wan. Thanks.